vm/actor/src/builtin/market/state.rs

```diff
@@ -245,7 +245,6 @@ impl State {
     // Deal state operations
     ////////////////////////////////////////////////////////////////////////////////
 
-    #[allow(dead_code)]
```

Comment:
See you removed some of these in this file, all should be removed in this whole folder, I just added them until this came in to avoid linting issues (can remove the TODO's on two of the files too about this)